eZ/Publish/Core/FieldType/User/Value.php

```diff
@@ -71,6 +71,11 @@ class Value extends BaseValue
      */
     public $maxLogin;
 
+    /**
+     * @var string
```

Comment:
```suggestion
     * @var string Write only property, takes a plain password for use when creating user or updating password.
```

Reply:
Maybe also a `@property-write` doc on class, but this is not a magic property so not a clear cut that it's relevant here, but it might help IDE's understand the purpose of the property.
ref: https://docs.phpdoc.org/references/phpdoc/tags/property-write.html